client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -322,13 +373,16 @@ class EditContactInfoFormCard extends React.Component {
 	}
 
 	saveContactInfo = ( event ) => {
-		event.preventDefault();
+		event.preventDefault && event.preventDefault();
```

Comment:
Yes - when this method is called from `renderDialog` when the user clicks `Confirm`, the `event` will be empty and this line would fail.